modules/contact-form/js/gutenblock.jsx

```diff
@@ -0,0 +1,568 @@
+/*global jQuery _ wp*/
```

Comment:
This file is really big and is doing a whole lot of stuff. I'd recommend breaking it into several files to make it easier to traverse, manage, and test.

Reply:
Probably agreed.  For the moment it's just grown as I was writing it and I'd love to break it up in a bit -- may swipe you for some help with that.